client/src/components/Map/index.js

```diff
@@ -69,6 +71,51 @@ function renderLearnMap(nodes, currentSuperBlock = '') {
         </li>
       ))}
     </ul>
+  ) : (
+    <ul data-test-label='learn-curriculum-map'>
+      {nodes
+        .filter(node =>
+          isAuditedCert(curriculumLocale, dasherize(node.superBlock))
+        )
+        .map((node, i) => (
+          <li key={i}>
+            <Link
+              className='btn link-btn btn-lg'
+              to={`/learn/${dasherize(node.superBlock)}/`}
+            >
+              <div style={linkSpacingStyle}>
+                {generateIconComponent(node.superBlock, 'map-icon')}
+                {createSuperBlockTitle(node.superBlock)}
+              </div>
+            </Link>
+          </li>
+        ))}
+      <hr />
```

Comment:
With the CTA at the bottom, I don't think it's clear what this `hr` indicates.  Could we put the text here?

For the text itself, how about

"The following certificates are still being translated.  Please go here(link to contribute docs) if you think you can help."

Reply:
I agree, we should move that up.

Reply:
I'll get this moved :)